model/search/filter/GlobalQueryFilter.java

```diff
@@ -36,7 +36,15 @@ public SearchSourceBuilder getSourceFilter() {
                                 createGlobalConditionalQueries(param, query))
                 );
         // Set Sort
-        if (query.getFilter().getDefaultSortField() !=null) builder.sort(query.getFilter().getDefaultSortField(), SortOrder.DESC);
+        if ("ASC".equalsIgnoreCase(query.getFilter().getSortDirection())){
```

Comment:
functionally this looks ok but it can be simplified and there is a lot of duplicated calls here. I would recommend storing the results of query.getFilter().getDefaultSortField() and query.getFilter().getSortDirection() in variables before the if statements. The logic here can also be simplified as shown below:

```
if default sort field is not null
    if sort direction is ASC
        sort ascending
    else
        sort descending
```